back_end/RayTracer.Tests/RayTracer.Tests.csproj

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Same point for these .csproj files - if they're autogenerated we don't want them in git...but I don't know :)

Reply:
We checked and this is necessary. Other package files were deleted